reference/OpenAPI.json

```diff
@@ -26857,571 +27429,1364 @@
           },
           {
-            "$ref": "#/components/schemas/9_obj_customer_activity_object_data_customer_voucher_gift_voucher_balance_added"
+            "$ref": "#/components/schemas/EventCustomerRewardRedemptionsCreated"
           },
           {
-            "$ref": "#/components/schemas/9_obj_customer_activity_object_data_customer_voucher_loyalty_card_points_added"
+            "$ref": "#/components/schemas/EventCustomerRewardRedemptionsPending"
           },
           {
-            "$ref": "#/components/schemas/9_obj_customer_activity_object_data_customer_voucher_loyalty_card_points_transferred"
+            "$ref": "#/components/schemas/EventCustomerRewardRedemptionsCompleted"
           },
           {
-            "$ref": "#/components/schemas/9_obj_customer_activity_object_data_customer_publication_succeeded"
+            "$ref": "#/components/schemas/EventCustomerRewardRedemptionsRolledBack"
           },
           {
-            "$ref": "#/components/schemas/9_obj_customer_activity_object_data_customer_publication_failed"
+            "$ref": "#/components/schemas/EventCustomerLoyaltyUpdated"
           },
           {
-            "$ref": "#/components/schemas/9_obj_customer_activity_object_data_customer_redemption_succeeded"
+            "$ref": "#/components/schemas/EventCustomerLoyaltyTierUpgraded"
           },
           {
-            "$ref": "#/components/schemas/9_obj_customer_activity_object_data_customer_redemption_failed"
+            "$ref": "#/components/schemas/EventCustomerLoyaltyTierDowngraded"
           },
           {
-            "$ref": "#/components/schemas/9_obj_customer_activity_object_data_customer_redemption_rollback_succeeded"
+            "$ref": "#/components/schemas/EventCustomerLoyaltyTierProlonged"
           },
           {
-            "$ref": "#/components/schemas/9_obj_customer_activity_object_data_customer_redemption_rollback_failed"
+            "$ref": "#/components/schemas/EventCustomerLoyaltyTierExpirationChanged"
           },
           {
-            "$ref": "#/components/schemas/9_obj_customer_activity_object_data_customer_consents_given"
+            "$ref": "#/components/schemas/EventCustomerLoyaltyTierJoined"
           },
           {
-            "$ref": "#/components/schemas/9_obj_customer_activity_object_data_customer_consents_revoked"
+            "$ref": "#/components/schemas/EventCustomerLoyaltyTierLeft"
+          }
+        ]
+      },
+      "EventCustomerSent": {
+        "title": "Event Customer Sent",
+        "type": "object",
+        "properties": {
+          "customer": {
+            "$ref": "#/components/schemas/SimpleCustomer"
+          },
+          "distribution": {
+            "type": "object"
+          },
+          "sent_at": {
+            "type": "string",
+            "example": "2021-12-22T10:13:06.487Z",
+            "description": "Timestamp representing the date and time when the sms was sent in ISO 8601 format.",
```

Comment:
To chyba nie będzie tylko SMS z tego co widzę. Bo jest użyte w kilku rodzajach dystrybucji.